engine/components/network/net.io/net.io-client/index.js

```diff
@@ -295,16 +295,19 @@ NetIo.Client = NetIo.EventingClass.extend({
 		// 	}
 		// }
 
+		// KILL SERVER MEANS THAT THE GAME WAS UNPUBLISHED ON STATE 3 AND STATE 2
```

Comment:
seems like we could add two different messages for states 3 and 2. Its kind of confusing still if the unpublish message could point to either.

Reply:
We don't need to be specific on the technical side of messages, what the user needs to know is that the game is not published on the current server anymore / connection lost

Reply:
Case 1 is for guest users

Reply:
Case 2 is for logged In users